openslides_backend/models/fields.py

```diff
@@ -465,5 +466,6 @@ def validate(self, value: Any, payload: Dict[str, Any] = {}) -> Any:
             return {key: sup.validate(struc) for key, struc in value.items()}
         elif type(value) == list:
             return value
-        else:
-            raise NotImplementedError
+        elif value is None:
+            return None
+        raise NotImplementedError
```

Comment:
Not form your changes, but noticing this just now: all the `raise NotImplementedError` lines are missing the brackets: `raise NotImplementedError()`